### PR TITLE
tests/e2e

### LINKED ISSUE #27: Add GitHub Action for Test Validation
Ensure all tests pass before building and deploying website to production. Use only unit and component tests (Do not use E2E tests).

### LINKED ISSUE #38: E2E Tests
* Create E2E tests with playwright
* Update docs to provide instructions on how to run

### PR DESCRIPTION
**Pull request by Green-Ranger11**
- [chore] rename to test:e2e instead
- [actions] add e2e test action
- [tests] add some simple e2e tests
- [actions] just run it everywhere

# Describe your changes

Simple e2e tests
## Issue ticket number and link
This closes #38 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Comment by Green-Ranger11** [comment]
Hold on let me add some instructions in `README`

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+concurrency:
+  # Use github.run_id on main branch
+  # Use github.event.pull_request.number on pull requests, so it's unique per pull request
+  # Use github.ref on other branches, so it's unique per branch
+  group: ${{ github.workflow }}-${{ github.ref == 'refs/heads/main' && github.run_id || github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  e2e:
+    timeout-minutes: 60
```
Comment (westerandr) [inline .github/workflows/tests.yml:16]: Are you able to make this 5 minutes please? Anything over, would be too long :)

**Comment by Green-Ranger11** [comment]
One sec please

**Comment by Green-Ranger11** [comment]
@westerandr let me know if you need any more changes

**Review by westerandr — APPROVED** [review]
LGTM! TY USO

**Pull request merged**